packages/webapp/src/inductions/components/induction-journey-container.tsx

```diff
@@ -18,14 +18,42 @@ export const INVITEE_INDUCTION_STEPS: Step[] = [
     },
     {
         title: "DONATE",
-        text: `${minimumDonation} to the Eden community.`,
+        text: `Give ${assetToString(
+            minimumDonationAmount
+        )} to the Eden community.`,
```

Comment:
Unless you really want to show 10.00000000 WAX, you can use `assetToString(minimumDonationAmount, 2)` -- which will limit in 2 decimals

Reply:
It has assetToString in there.

Reply:
Oh, the 2! Missed that. Thank you!

Reply:
`assetToString` defaults to 2 decimals. (And it's already truncating properly.)